README.md

```diff
@@ -135,6 +135,37 @@ For each filter, AdGuard compiles two versions: full and optimized. Optimized ve
 
 `custom_platforms.js` customizes the way filters are compiled for certain platforms. We should use it if we need to
 temporary change rules for a platform. In all other cases, we should prefer the default configuration.
+Below is a example of the configuration for the platform `AdGuard for Chrome` with comments:
+```javascript
+    "EXTENSION_CHROMIUM": {
+        "platform": "ext_chromium", // Defines the platform for which the settings are specified.
```

Comment:
single line comments should be placed above the subject of the comment, not in the same line
https://github.com/AdguardTeam/CodeGuidelines/blob/master/JavaScript/Javascript.md#comments--singleline

Reply:
Done.